pacemaker.spec.in

```diff
@@ -355,6 +355,10 @@ install -m 644 mcp/pacemaker.combined.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/in
 install -m 644 tools/crm_mon.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/init/crm_mon.conf
 %endif
 
+%if %{defined _unitdir}
+mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/lib/rpm-state/%{name}
+%endif
```

Comment:
btw. tried multiple owning of the %ghosted directory and it is working as expected:
after removing the last packet owning the %ghosted directory it is properly removed
Thought the idea was having rpm do the counting for the subdir and the flag-files,
which are just used by one of the pacemaker-subpackages can be handled explicitly.